first_lessons_dushanbe/array_sorting_bubbles.py

```diff
@@ -6,14 +6,14 @@ def list_sort(a):
     if a is None:
         return a
     count = len(a)-1
-    while count > 1:
+    while count >= 1:
         ln = 0
         for ln in range(count):
             if a[ln] > a[ln+1]:
-                (a[ln+1], a[ln]) = (a[ln], a[ln+1])
+                (a[ln], a[ln + 1]) = (a[ln + 1], a[ln])
```

Comment:
Could you please remove unnecessary `()?

Reply:
done